.github/workflows/ci.feature.yml

```diff
@@ -105,6 +107,90 @@ jobs:
         run: |
           yarn workspace @superfluid-finance/ethereum-contracts test-coverage
 
+
+  #test automations:
+  test-automation-contracts-scheduler:
+    name: Build and test automation-contracts-scheduler (Feature Branch)
+
+    runs-on: ubuntu-latest
+
+    needs: [ check ]
+    if: needs.check.outputs.build_automation_contracts_scheduler
+
+    strategy:
+      matrix:
+        node-version: [ 16.x ]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Check changeset
+        run: tasks/check-changeset.sh ${{ github.sha }} dev
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "yarn"
+
+      - name: Install, lint and build
+        run: |
+          yarn install --frozen-lockfile
+          yarn lint
+          yarn build
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: Test automation-contracts-scheduler
+        run: |
+          yarn workspace scheduler test
+
+  test-automation-contracts-autowrap:
+    name: Build and test automation-contracts-autpwrap (Feature Branch)
+
+    runs-on: ubuntu-latest
+
+    needs: [ check ]
+    if: needs.check.outputs.build_automation_contracts_autowrap
+
+    strategy:
+      matrix:
+        node-version: [ 16.x ]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Check changeset
```

Comment:
please delete this step, it is duplicated and unused in the job, I forgot to remove this from the ethereum-contracts job.